guides/v2.2/config-guide/config/disable-module-output.md

```diff
@@ -7,31 +7,33 @@ functional_areas:
   - Setup
 ---
 
-By default, all modules are configured so that a module's output can be written to a view. Turning off output offers a way to essentially disable a module that can't be disabled due to hard dependencies.
+By default, all modules are configured so that a module's output can be written to a view. Turning off output offers a way to essentially disable a module that can not be disabled due to hard dependencies.
 
-For example, the `Customer` module depends on the `Review` module, so the `Review` module can't be disabled. However, if you don't want customers to be able to provide reviews, you could turn off output from the `Review` module.
+For example, the `Customer` module depends on the `Review` module, so the `Review` module can not be disabled.
+However, if you do not want customers to be able to provide reviews, you could turn off output from the `Review` module.
 
 {:.bs-callout .bs-callout-info}
 If a merchant used the Admin to disable a module's output in a previous release, you must manually configure the system to migrate these settings.
 
 ## Disable module output in a pipeline deployment
 
-To disable module output in pipeline or any other deployment with multiple instances of Magento:.
+To disable module output in pipeline or any other deployment with multiple instances of Magento:
```

Comment:
```suggestion
To disable module output in the pipeline, or any other deployment, with multiple instances of Magento:
```

Reply:
I would say "in a pipeline deployment or in any other deployment..."

Reply:
Fixed.